pytorch_pfn_extras/handler/_logic.py

```diff
@@ -273,6 +274,16 @@ def train_step(
             batch (torch.Tensor, list of torch.Tensor, dict of torch.Tensor):
                 Input tensors feeded to the model of the current step.
         """
+        module = list(models.values())[0]
+        optimizer = list(optimizers.values())[0]
+
+        return update_parameters(
+            module,
+            optimizer,
+            None,  # backprop_from should be an opt
+            None,
+        )(batch)
+
         with torch_autocast(enabled=self._autocast):
```

Comment:
these statement is not reachable because return in line 280.

Reply:
Yeah, just keeping old code around for refactoring, still wip.
Thanks!